cmd/common/snapshot/process.go

```diff
@@ -64,36 +63,19 @@ func (p *ProcessParser) GetOutputConfig() *commonutils.OutputConfig {
 	}
 }
 
-func (p *ProcessParser) SortEvents(allProcesses *[]types.Event) {
+func (p *ProcessParser) SortEvents(allProcesses []*types.Event) {
```

Comment:
This change is reverted in the next commit. Can you make the right change in this commit to avoid changing it back in the next one?